plugins/automation/admin/assets/javascripts/admin/components/fields/da-groups-field.gjs

```diff
@@ -0,0 +1,55 @@
+import { tracked } from "@glimmer/tracking";
+import { hash } from "@ember/helper";
+import { action } from "@ember/object";
+import Group from "discourse/models/group";
+import GroupChooser from "select-kit/components/group-chooser";
+import BaseField from "./da-base-field";
+import DAFieldDescription from "./da-field-description";
+import DAFieldLabel from "./da-field-label";
+
+export default class GroupsField extends BaseField {
+  @tracked allGroups = [];
+
+  constructor() {
+    super(...arguments);
+
+    Group.findAll({
```

Comment:
Can we use `this.site.groups` here instead of making a request to retrieve the list of groups?

Reply:
Happy to do a follow up to handle this, but how do you handle `this.args.field.extra.ignore_automatic` in this case?

Reply:
Oh, right, the site serializer doesn't include the `automatic` field of groups:

https://github.com/discourse/discourse/blob/d642020b0fee8a847b89f0a9ae382297404ec034/app/serializers/site_serializer.rb#L94-L111

I think it'd be reasonable and safe to include `automatic` in there, but I agree that can be done in a follow-up PR.